wm/property.go

```diff
@@ -158,3 +158,11 @@ func windowExtendedHintsRemove(x *xgbutil.XUtil, win xproto.Window, hint string)
 		}
 	}
 }
+
+func windowClientListSet(x *xgbutil.XUtil, wins []xproto.Window) {
```

Comment:
Because the functions are in the property.go file as they are ewmh settings and it follows the convention of every single function in propoerty.go starting with window.  We can revisit that?